src/lang/english.txt

```diff
@@ -935,7 +935,6 @@ STR_GAME_OPTIONS_CURRENCY_MXN                                   :Mexican Peso (M
 STR_GAME_OPTIONS_CURRENCY_NTD                                   :New Taiwan Dollar (NTD)
 STR_GAME_OPTIONS_CURRENCY_CNY                                   :Chinese Renminbi (CNY)
 STR_GAME_OPTIONS_CURRENCY_HKD                                   :Hong Kong Dollar (HKD)
-STR_GAME_OPTIONS_CURRENCY_INR                                   :Indian Rupee (INR)
```

Comment:
Why did you remove this line? #8136 added it.

Reply:
James103, my bad. I'll fix that.